# Assignment \#8 : The Astrophysical Distance Ladder

Kaimi Kahihikolo

Due: 23 April 2019

In [1]:
import astropy.units as u
import astropy.constants as const
import numpy as np

___
## 1 Core-Collapse Supernovae

At the end of its life, a massive star has contracted from its original molecular cloud to become a neutron star with a mass of 1.5 times the solar mass and a radius of 13 km. 

- (1.1) Calculate the total gravitational energy that has been released in forming the neutron star. 

- (1.2) Assume that the star had a constant luminosity of 35,000 times the solar luminosity for the 6 million years of its entire life (from zero-age main sequence to final collapse and explosion). 

- (1.3) Having previously spent all that luminous energy throughout its lifetime, does the exploding star have enough energy left to eject 12 solar masses of stellar material with an expansion velocity of 10,000 km/s? Show your calculations; do not just reply yes or no.

**(1.1)** Recall, the potential energy,

$$U = \frac{GM^2}{R}$$

In [44]:
M_N = 1.5 * u.Msun
R_N = 13 * u.km

U = const.G * M_N**2 / R_N
print(f"\t U = {U.cgs:.2e}")

	 U = 4.57e+53 erg


**(1.2)** Over the star's lifetime, it release 35,000 $L_\odot$ over the period of $6\times 10^6$ years. Thus, the energy it releases:

In [45]:
L_N = 35000 * const.L_sun
T = 6e6 *u.year

E_released = (L_N * 6e6 * u.year).cgs
print(f"\t E_released = {E_released.cgs:.2e}")

	 E_released = 2.54e+52 erg


**(1.2)** After losing all that luminous energy, the energy required to eject $12 M_\odot$ is given by, $\frac{1}{2} m v^2$.

In [43]:
v = 10e3 *u.km /u.s
M_eject = 12 * u.Msun

E_eject = 0.5 * M_eject * v**2
print(f"\t E_eject = {E_eject.cgs:.2e}")

	 E_eject = 1.19e+52 erg


Since the total energy is approximately $4\times 10^{53}$ [erg], after releasing about $2.5 \times 10^{52}$ [erg] in luminous energy, there is still sufficient energy to eject the 12 solar masses of material.

___
## (2) Escape Velocities and Fast Rotations

- **(2.1)** The calculations we will make here are part of the reasoning that leads to the identification of pulsars as rapidly rotating neutron stars. Calculate the following escape velocities in km/s: 
    - (a) from the surface of a white dwarf with the same mass as the Sun, but the same radius as the Earth.
    - (b) from the surface of a neutron star with 1.5 times the mass of the Sun and a radius of 13 km.

- **(2.2)** Consider how fast the Earth rotates. The circumference at the Earth’s equator is 40,000 km, and the rotation takes one day, so our rotation speed at the equator is about 0.5 km/s. OK, now imagine a white dwarf with one solar mass and the size of the Earth. Imagine it spins so fast that it makes 2 rotations per second. Calculate how fast the surface of the white dwarf is moving at its equator, in km/s. Compare this speed with the escape velocity from the surface, which you calculated earlier, and explain what would happen to your fast rotating white dwarf.

- **(2.3)** Now repeat your calculation, using the neutron star you considered earlier, but assuming it makes 100 rotations per second. Is the rotation speed at the equator larger than the escape velocity? How fast do you think this neutron star could rotate? In other words, how many rotations per second, maximum? Find in the literature what is the shortest known pulsar period, and compare with your result.